### PR TITLE
Add method to reset Media folder

### PR DESCRIPTION
**Pull request by NickAb**

### CODE REVIEW

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    udid: UDID,
+    deviceSetPath: String
+) {
+    private val mediaPath = Paths.get(deviceSetPath, udid, "data", "Media")
+
+    fun reset() {
+        val imagesPath = mediaPath.resolve("DCIM").toString()
+        val photoDataPath = mediaPath.resolve("PhotoData/Photos.sqlite").toString()
+
+        val shellCmd = "rm $imagesPath/**/* $photoDataPath*; touch $photoDataPath"
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt]: what if there isn't any ? how about make it 
```
rm -f
```

**Review by fr0l — APPROVED** [review]

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    private fun restartAssetsd() {
+        val restartCmd = listOf(
+            "xcrun", "simctl", "spawn", udid, "launchctl", "kickstart", "-k", "-p", "system/com.apple.assetsd"
+        )
+
+        val result = remote.execIgnoringErrors(restartCmd)
+
+        if (!result.isSuccess) {
+            throw RuntimeException("Could not restart assetsd service")
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt]: let's print stderr here as well

**Review by fr0l — CHANGES_REQUESTED** [review]

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    private val udid: UDID,
+    deviceSetPath: String
+) {
+    private val mediaPath = Paths.get(deviceSetPath, udid, "data", "Media")
+
+    fun reset() {
+        val imagesPath = mediaPath.resolve("DCIM").toString()
+        val photoDataPath = mediaPath.resolve("PhotoData/Photos.sqlite").toString()
+
+        val removeCmd = "rm $imagesPath/**/* $photoDataPath*; touch $photoDataPath"
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt]: how about using rm -f
here

**Review by fr0l — APPROVED** [review]

**Comment by NickAb** [comment]
If we will need to list available media from the gallery or delete specific photos from gallery instead of wiping it out completely, we can take a look at https://github.com/geiszla/iOSLib for reference.

**Pull request merged**